tests/conftest.py

```diff
@@ -12,16 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import click
+import pytest
 
-from pioinstaller import __title__, __version__
+from pioinstaller.pack import packer
 
 
-@click.group()
-@click.version_option(__version__, prog_name=__title__)
-def cli():
-    pass
-
-
-def main() -> int:
-    return cli()
+@pytest.fixture(scope="session")
+def pio_installer_script():
+    packer.pack()
```

Comment:
Should return a path to `get-platformio.py`. Please use `tmpdir` from pytest and pack get-platformio to this `dst` dir